Actors/Assets/PlayerActor.shader

```diff
@@ -0,0 +1,43 @@
+shader_type spatial;
```

Comment:
This stuff doesn't fit in the UI changes PR, try to make branches for whatever you are experimenting with, you can have more than one going no problem.